librecad/src/lib/math/rs_math.h

```diff
@@ -36,117 +36,122 @@ class QString;
-    @ce, a vector of size 4 contains the coefficient in order
-    @return, a vector contains real roots
-    **/
-    static std::vector<double> quarticSolver(const std::vector<double>& ce);
-    /** quartic solver
-* ce[4] x^4 + ce[3] x^3 + ce[2] x^2 + ce[1] x + ce[0] = 0
-    @ce, a vector of size 5 contains the coefficient in order
-    @return, a vector contains real roots
-    **/
-    static std::vector<double> quarticSolverFull(const std::vector<double>& ce);
-    //solver for linear equation set
-    /**
-      * Solve linear equation set
-	  *@param mt holds the augmented matrix
-	  *@param sn holds the solution
-	  *@param return true, if the equation set has a unique solution, return false otherwise
-      *
-	  *@author: Dongxu Li
-      */
-	static bool linearSolver(const std::vector<std::vector<double> >& m, std::vector<double>& sn);
-
-    /** solver quadratic simultaneous equations of a set of two **/
-    /* solve the following quadratic simultaneous equations,
-      *  ma000 x^2 + ma011 y^2 - 1 =0
-      * ma100 x^2 + 2 ma101 xy + ma111 y^2 + mb10 x + mb11 y +mc1 =0
-      *
-      *@m, a vector of size 8 contains coefficients in the strict order of:
-      ma000 ma011 ma100 ma101 ma111 mb10 mb11 mc1
-      *@return a RS_VectorSolutions contains real roots (x,y)
-      */
-    static RS_VectorSolutions simultaneousQuadraticSolver(const std::vector<double>& m);
-
-    /** solver quadratic simultaneous equations of a set of two **/
-	/** solve the following quadratic simultaneous equations,
-      * ma000 x^2 + ma001 xy + ma011 y^2 + mb00 x + mb01 y + mc0 =0
-      * ma100 x^2 + ma101 xy + ma111 y^2 + mb10 x + mb11 y + mc1 =0
-      *
-  *@param m a vector of size 2 each contains a vector of size 6 coefficients in the strict order of:
-  ma000 ma001 ma011 mb00 mb01 mc0
-  ma100 ma101 ma111 mb10 mb11 mc1
-      *@return a RS_VectorSolutions contains real roots (x,y)
-      */
-    static RS_VectorSolutions simultaneousQuadraticSolverFull(const std::vector<std::vector<double> >& m);
-    static RS_VectorSolutions simultaneousQuadraticSolverMixed(const std::vector<std::vector<double> >& m);
-
-	/** \brief verify simultaneousQuadraticVerify a solution for simultaneousQuadratic
-	  *@param m the coefficient matrix
-	  *@param v a candidate to verify
-      *@return true, for a valid solution
-      **/
-	static bool simultaneousQuadraticVerify(const std::vector<std::vector<double> >& m, RS_Vector& v);
-    /** wrapper for elliptic integral **/
-    /**
-     * wrapper of elliptic integral of the second type, Legendre form
-     *@k the elliptic modulus or eccentricity
-     *@phi elliptic angle, must be within range of [0, M_PI]
-     *
-	 *@\author: Dongxu Li
-     */
-    static double ellipticIntegral_2(const double& k, const double& phi);
-
-    static QString doubleToString(double value, double prec);
-    static QString doubleToString(double value, int prec);
-
-    static void test();
+class RS_Math
+{
```

Comment:
Most of these changes are just whitespace adjustments to align the code and make it presentable.